src/deploy/osp_deployer/director.py

```diff
@@ -553,7 +558,15 @@ def setup_environment(self):
                     logger.warning(
                         "Bad entry in osd_disks: {}".format(osd))
 
+        if osds_per_node < 1:
```

Comment:
I don't think we can have this check (561-563) because it's perfectly ok to omit the OSD configuration from the .properties file if you have 14g storage nodes with BOSS+HBA330 since we auto-generate the OSD config in this case.

Reply:
I removed it.